src/main/java/com/xiaomi/infra/pegasus/client/ClientOptions.java

```diff
@@ -58,9 +58,7 @@
   public static final Duration DEFAULT_FALCON_PUSH_INTERVAL = Duration.ofSeconds(10);
   public static final boolean DEFAULT_ENABLE_WRITE_LIMIT = true;
   public static final Duration DEFAULT_META_QUERY_TIMEOUT = Duration.ofMillis(5000);
-  public static final boolean DEFAULT_ENABLE_AUTH = false;
```

Comment:
delete `DEFAULT_ENABLE_AUTH`, how judge open or close? `DEFAULT_AUTH_PROTOCOL = ""` means close?

Reply:
yes, you are right